Code/Core/Env/Env.cpp

```diff
@@ -221,8 +221,10 @@ void Env::GetExePath( AString & output )
         output = argv[0];
     #else
         char path[ PATH_MAX ];
-        VERIFY( readlink( "/proc/self/exe", path, PATH_MAX ) != -1 );
-        output = path;
+        const ssize_t length = readlink( "/proc/self/exe", path, PATH_MAX );
+        VERIFY( length != -1 );
+        VERIFY( length <= PATH_MAX );
```

Comment:
These VERIFY statements should be ASSERTs. I'll fix that post-merge.
ASSERT = stripped in Release builds
VERIFY = code remains, but assertion of result is stripped in Release builds.